### PR TITLE
Add AADE Foundation to Tidepool Big Data Donation Project Nonprofit Partner List

### PR DESCRIPTION
**Pull request by courtenayhuffman**
See: https://trello.com/c/LWBzvGMr

### CODE REVIEW

**Comment by courtenayhuffman** [comment]
[Add AADE Foundation to Tidepool Big Data Donation Project Nonprofit Partner List](https://trello.com/c/LWBzvGMr/397-add-aade-foundation-to-tidepool-big-data-donation-project-nonprofit-partner-list)

File: app/core/constants.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ export const DATA_DONATION_NONPROFITS = [
   { value: 'JDRF', label: 'JDRF' },
   { value: 'NSF', label: 'Nightscout Foundation' },
   { value: 'T1DX', label: 'T1D Exchange' },
+  { value: 'AADE', label: 'American Association of Diabetes Educators' },
```
Comment (cbwebdevelopment) [inline app/core/constants.js]: I see that the trello card description shows the name as `AADE Foundation`, which I imagine is likely what was intended to be used as the label.

File: app/core/constants.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ export const DATA_DONATION_NONPROFITS = [
   { value: 'JDRF', label: 'JDRF' },
   { value: 'NSF', label: 'Nightscout Foundation' },
   { value: 'T1DX', label: 'T1D Exchange' },
+  { value: 'AADE', label: 'American Association of Diabetes Educators Foundation' },
```
Comment (cbwebdevelopment) [inline app/core/constants.js]: What I meant in the previous change request was that we want the label to be `AADE Foundation` exactly (i.e. not expanding the `AADE` abbreviation).

**Comment by cbwebdevelopment** [comment]
LGTM! 👍

**Pull request merged**